src/components/tl/mlx5/tl_mlx5_coll.h

```diff
@@ -23,15 +28,15 @@ typedef struct ucc_tl_mlx5_schedule {
 } ucc_tl_mlx5_schedule_t;
 
 #define TASK_TEAM(_task)                                                       \
-    (ucc_derived_of((_task)->super.super.team, ucc_tl_mlx5_team_t))
+    (ucc_derived_of((_task)->super.team, ucc_tl_mlx5_team_t))
```

Comment:
Ok I see, I guess this change makes sense.
So these macros were written to take `ucc_tl_mlx5_schedule_t` as an argument. With this patch, they now take the parent class, `ucc_tl_mlx5_task_t`.
I'll change the a2a code accordingly